modules/checking-mco-status.adoc

```diff
@@ -23,11 +23,12 @@ where:
 +
 --
 ** `UPDATED` indicates that the MCO applied the current machine config to the nodes in that MCP.
-** `UPDATING` indicates that the MCO is applying the current machine config to the nodes in that MCP. Nodes that are in `UPDATING` status are not available for scheduling.
-** `DEGRADED` indicates the MCO failed to apply the current machine config to the nodes in that MCP. Nodes that are in `DEGRADED` status are not available for scheduling.
+** `UPDATING` indicates that the MCO is applying the current machine config to at least one of the nodes in that MCP. Nodes that are in `UPDATING` status are not available for scheduling.
```

Comment:
So, `UPDATING` and `DEGRADED` nodes are not necessarily "not available for scheduling". Those are separate states, so maybe we should either remove that line, or say something like ```Nodes that are in `UPDATING` status may not be available for scheduling.```

Reply:
Done!

Reply:
That's a bit harder to answer actually. Degraded nodes are **most likely** not available for scheduling, which some rare cases where it could be.

I don't mind keeping it either way. Degraded nodes indicate a problem so its fine to keep as is